SpatialGDK/Source/SpatialGDKServices/Public/EditorWorkerController.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
Can you make the relative public/private paths match? (i.e put EditorWorkerController.h inside Interop/Connection)

Reply:
Moved.